istioctl/cmd/istioctl/istioctl_test.go

```diff
@@ -189,14 +179,14 @@ func TestGet(t *testing.T) {
 			configs: testGateways,
 			args:    strings.Split("get gateways -n default", " "),
 			expectedOutput: `GATEWAY NAME       HOSTS     NAMESPACE   AGE
-bookinfo-gateway   *         default     <unknown>
```

Comment:
Nit: Would it ever be possible that this test might fail if it takes >1 second to execute?  I'm concerned that "0s" might turn into "1s".

Reply:
Yeah it would fail in that case.  Should be *very* unlikely, though.